UI/GameInfoCache.cpp

```diff
@@ -720,6 +720,7 @@ std::shared_ptr<GameInfo> GameInfoCache::GetInfo(Draw::DrawContext *draw, const
 	if (info->IsWorking()) {
 		// Uh oh, it's currently in process.  It could mark pending = false with the wrong wantFlags.
 		// Let's wait it out, then queue.
+		// NOTE: This is bad because we're likely on the UI thread....
```

Comment:
Hmm, yeah.  It should be uncommon.  The other option is to make it so we can enqueue another item and auto-delete this one somehow.  We just don't want it marking itself as done/ready when it started off the wrong flags.

This used to cause race conditions.

-[Unknown]